pkg/kubelet/pod_workers.go

```diff
@@ -28,7 +28,15 @@ import (
 	"github.com/golang/glog"
 )
 
-type syncPodFnType func(*api.Pod, *api.Pod, kubecontainer.Pod) error
+const (
+	minBackoffInterval = time.Second * 4
```

Comment:
Would it be useful to express these in terms of the sync interval since that is how often we'll restart?